src/Finelines/Stages/YamlStage.fs

```diff
@@ -21,6 +22,10 @@ type YamlStage with
             this.DisplayName
             |> Option.map (fun name -> $"displayName: {format (Text name)}")
 
+        let condition = 
+            this.Condition
+            |> Option.map (fun condition -> $"condition: {format (Text condition)}")
```

Comment:
One problem with using `Text` is that the result gets wrapped in single quotes, which produces invalid yaml:
```
condition: 'and(succeeded(), in(variables['Build.Reason'], 'IndividualCI'))'
```

My guess is that we would like to extend the `Input` DU with a `Condition` or `Expression` or `ConditionExpression` case, that would never wrap the string value in quotes in `Format.fs`, and use it here:
```
let condition = 
    this.Condition
    |> Option.map (fun condition -> $"condition: {format (ConditionExpression condition)}")
```